mpp/mpp.cpp

```diff
@@ -626,6 +638,9 @@ MPP_RET Mpp::control_mpp(MpiCmd cmd, MppParam param)
         MppPollType block = *((MppPollType *)param);
         mOutputBlock = block;
     } break;
+    case MPP_SET_OUTPUT_BLOCK_TIMEOUT: {
+        mOutputBlockTimeout = *((int *)param);
```

Comment:
BTW, the error handling can be more general. I do not think carefully about the error code design. If you have any suggestion do not be hesitated to add a commit : )